modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -73,6 +73,62 @@ spec:
       rootCA: ""
 ----
 
+* Optional: Customize the route name of the Keycloak instance by adding a host attribute in the Argo CD CR. Use this feature to support advanced routing use cases such as balancing incoming traffic load among multiple link:https://docs.openshift.com/container-platform/latest/networking/ingress-sharding.html#nw-ingress-sharding_ingress-sharding[Ingress Controller shards].
++
+** To add a `host` attribute in the Argo CD CR, add this YAML snippet in your namespace.
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: route
+spec:
+  sso:
+    provider: keycloak
+    keycloak:
+     host: sso.test.example.com #<1>
+  server:
+    ingress:
+      enabled: true
+    insecure: true
+----
+<1> Specifies the name of the host key
++
+** To apply the YAML, run the following command in the {gitops-shortname} CLI:
++
+[source,terminal]
+----
+oc apply -f argocd.yaml -n <any-namespace>
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values

```suggestion
oc apply -f <argocd_filename>.yaml -n <your_namespace>
```

There is no instruction or info that the argocd CR is being saved as `argocd.yaml`. Hence it's potentially a userpreplaced variable.

Reply:
Yes, I think the user will have a namespace being created.

Reply:
Fixed.